i18n/en/code.json

```diff
@@ -569,5 +596,8 @@
   },
   "theme.docs.breadcrumbs.home": {
     "message": "Home page"
+  },
+  "fabric.subjeting": {
```

Comment:
Can we re-use "home_page.edition_ms_fabric": {
    "message": "Microsoft Fabric"
  } key, wherever needed instead of adding the same translation again.

Reply:
Ok